lib/controller/StudyRequestBulkController.js

```diff
@@ -0,0 +1,92 @@
+import Boom from '@hapi/boom';
+
+import { AuthScope } from '@/lib/Constants';
+import StudyRequestBulkDAO from '@/lib/db/StudyRequestBulkDAO';
+import StudyRequestBulk from '@/lib/model/StudyRequestBulk';
+import EmailStudyRequestBulkConfirmation from '@/lib/email/EmailStudyRequestBulkConfirmation';
+import Mailer from '@/lib/email/Mailer';
+import LogTag from '@/lib/log/LogTag';
+import Joi from '@/lib/model/Joi';
+
+/**
+ * CRUD handling for bulk study requests, including actions that affect all
+ * sub-requests.
+ *
+ * @type {Array<Hapi.ServerRoute>}
+ */
+const StudyRequestBulkController = [];
+
+/**
+ * Create a new bulk study request.  This endpoint also sends an email to the requester
+ * and any persons CC'd.
+ *
+ * The request body should contain the bulk request, including any studies requested
+ * therein, in JSON format.
+ *
+ * @memberof StudyRequestBulkController
+ * @name postStudyRequestBulk
+ * @type {Hapi.ServerRoute}
+ */
+StudyRequestBulkController.push({
+  method: 'POST',
+  path: '/requests/study/bulk',
+  options: {
+    auth: {
+      scope: [AuthScope.STUDY_REQUESTS_EDIT.name],
+    },
+    response: {
+      schema: StudyRequestBulk.read,
+    },
+    validate: {
+      payload: StudyRequestBulk.create,
+    },
+  },
+  handler: async (request) => {
+    const user = request.auth.credentials;
+    const studyRequestBulk = await StudyRequestBulkDAO.create(request.payload, user);
+    try {
+      const email = new EmailStudyRequestBulkConfirmation(user, studyRequestBulk);
+      const emailOptions = await email.getOptions();
+      const emailResponse = await Mailer.send(emailOptions);
+      request.log(LogTag.DEBUG, emailResponse);
+    } catch (err) {
+      request.log(LogTag.ERROR, err);
```

Comment:
can we log the `err` in `StudyRequestController` as well?  this will be _very_ useful for debugging.